protocols/v2/binary-sv2/no-serde-sv2/derive_codec/src/lib.rs

```diff
@@ -1,4 +1,13 @@
+#![no_std]
```

Comment:
why aren't we following this pattern here?
```
#![cfg_attr(feature = "no_std", no_std)]
```

Reply:
Same: KISS.